src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -36,6 +36,103 @@ public static partial class Kernel32
         /// </summary>
         public const int NMPWAIT_NOWAIT = 0x00000001;
 
+        /// <summary>Hardware-dependent cursor resource.</summary>
+        public static readonly unsafe char* RT_CURSOR = MAKEINTRESOURCE(1);
```

Comment:
I don't know, moving them to somewhere that isn't referenced when doing an `using static Kernel32` would make them a lot harder to discover especially as they don't have a type for themselves that could act as a good storage location for special values.

But we could move all resources related functions and constants to a separate `partial` part, it won't ease consumption but can make the library code more readable.